example/js_components/js_components.dart

```diff
@@ -0,0 +1,79 @@
+@JS()
+library js_components;
+
+import 'dart:html';
+import 'dart:math';
+
+import 'package:js/js.dart';
+import 'package:react/react.dart' as react;
+import 'package:react/react_client.dart';
+import 'package:react/react_client/react_interop.dart';
+import 'package:react/react_dom.dart' as react_dom;
+
+var fooRef;
+
+main() {
```

Comment:
@greglittlefield-wf is this file sufficient to document / provide OSS explanation(s) for #109? Or should an example be created that shows how to "wrap" an existing JS component from a library like material-ui?

Reply:
Updated this example to use Material UI and A custom component

Reply:
It looks like that is sufficient! 🏅